easybuild/easyconfigs/r/ReFrame/ReFrame-2.18.eb

```diff
@@ -0,0 +1,26 @@
+easyblock = 'PythonPackage'
+
+name = 'ReFrame'
+version = '2.18'
+
+homepage = 'https://github.com/eth-cscs/reframe'
+description = '''ReFrame is a framework for writing regression tests for HPC systems.'''
+
+toolchain = {'name': 'dummy', 'version': ''}
+
+source_urls = ['https://pypi.python.org/packages/source/R/ReFrame-HPC']
+sources = ['ReFrame-HPC-%s.tar.gz' % version]
```

Comment:
@boegel I tried to use `%(version)s` here but didn't work, so I reverted to the standard way.

Reply:
There's absolutely no reason why it shouldn't work... How did it fail exactly?

```python
sources = ['ReFrame-HPC-%(version)s.tar.gz']

Reply:
It works. I don't know what I've done wrong before. I fixed that.